### DISCUSSION TITLE
Swag kits

### DISCUSSION THREAD

**Discussion by hmuhammadazeem** [community/community#126229] [GitHub Education]
Hi, I'd received GitHub Swag in 2022 that included Git printed booklets, stickers and T-Shirts vouchers. I'd distributed the booklets among students. But I left my teaching position since, and these stickers have been still lying around in my drawer. What to do with them? 

1. Throw away 
2. Give them away to kids around me? 

BTW the vouchers got expired.

Best.

**Comment by AsmitaMishra24** [comment]
Give them away to kids around you
